### DISCUSSION TITLE
Make breaktime overlay not appear if there are notes during the breaktime

### DISCUSSION THREAD

**Discussion by RubiksImplosion** [ppy/osu#27457] [Ideas] [answered]
Currently this is what happens when there are notes during the breaktime:

video:
https://github.com/ppy/osu/assets/39072130/fcb9dcee-4182-4930-902d-1da705cea3a9

map in video: 
https://osu.ppy.sh/beatmapsets/710687#mania/1502424

The obscuring of the playfield while there are notes on screen is extremely distracting.
In stable, the background undims itself and the leaderboards show up, but nothing actively obscures the playfield.

I propose that the breaktime overlay should not be shown if there are any notes during the breaktime, even if the beatmap file says a break should be present there.

**Comment by peppy** [comment]
Notes during break time sounds like a broken beatmap.

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/10798.
